.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,99 @@
+name: Pre-release tests
+
+on:
+  # Runs at 5:00 UTC on Mondays
+  schedule:
+    - cron: "0 5 * * 1"
+  workflow_dispatch:
+
+jobs:
+
+  make-check-job:
+    name: Run 'make check'
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: pip install -r requirements-dev.txt
+      - name: Installing Avocado in develop mode
+        run: python3 setup.py develop --user
+      - name: Run make check
+        run: make check
+
+  avocado-pre-release-job:
+    name: Avocado pre-release job
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install avocado
+        run: |
+          sudo python3 -m pip install -r requirements-dev.txt
+          sudo python3 setup.py develop
+      - name: Avocado pre-release job
+        run: ./selftests/pre_release/jobs/pre_release.py
+
+
+  # Re-implements ./selftests/pre_release/tests/check-copr-rpm-version.sh
+  check-rpm-available-copr-job:
+    name: Check the right RPM packages are available in COPR
+    runs-on: ubuntu-latest
+    env:
+      FEDORA_VERSION: '34'
+    container:
+      image: fedora:34
+    steps:
+      - name: install repos and packages
+        run: |
+          dnf -y install git
+          dnf -y module disable avocado
+          dnf -y install 'dnf-command(copr)'
+          dnf -y copr enable @avocado/avocado-latest
```

Comment:
Out of curiosity: why sudo is not needed here, but needed on lines 38-39?

Reply:
@beraldoleal Here it's running inside of the` fedora:34` container and on lines 38-39 it's running inside the ubuntu VM as regular user so you need sudo to install avocado system-wide.